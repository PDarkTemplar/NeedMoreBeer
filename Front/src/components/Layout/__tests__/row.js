// @flow
import { shallow } from 'enzyme';

import React from 'react';

import { Row, Cell } from '../index';

test('Should render row', () => {
    const row = shallow(
        <Row>
            <Cell size={12}>test</Cell>
        </Row>
    );

    expect(row).toMatchSnapshot();
});

test('Should render row (align start)', () => {
    const row = shallow(
        <Row align="start">
            <Cell size={12}>test</Cell>
        </Row>
    );

    expect(row).toMatchSnapshot();
});
