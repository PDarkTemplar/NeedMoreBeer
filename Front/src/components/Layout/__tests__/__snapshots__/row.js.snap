// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render row (align start) 1`] = `
<div
  className="row"
>
  <Cell
    size={12}
  >
    test
  </Cell>
</div>
`;

exports[`Should render row 1`] = `
<div
  className="row"
>
  <Cell
    size={12}
  >
    test
  </Cell>
</div>
`;
